RR.UnitOfWork.Tests/Entities/EmployeeCertificationUnitTests.cs

```diff
@@ -28,7 +28,8 @@ public EmployeeCertificationUnitTests()
             IssueDate = DateTime.Now,
             IssueOrganization= "String",
             CertificateDocument = "as",
-            CertificateName = "Name"
+            CertificateName = "Name",
+            DocumentName = "hello"
```

Comment:
Refactor: The readability of the strings could be improved here.

CertificateName = "CertificateName"
DocumentName = "DocumentName"